### PR TITLE
Clarify pointercancel coordinates

### LINKED ISSUE #463: Coordinates of a `pointercancel` event
I think the spec should say that the coordinates of a `pointercancel` event are uninitialized (or zeros).

Chrome does not initialize the coordinates of a `pointercancel` event, which seems logical because the event is tied to a specific `pointerId` and not related to a specific pointer location.  For example, if a touch-drag causes scroll (and therefore the firing of a `pointercancel` event), the user agent absorbs the last `pointermove` events to determine the scroll amount etc and the page is not expected to know those event locations.  Developers can cache the last `pointermove` for that `pointerId` if there is any use.

We have a [Chrome bug](https://bugs.chromium.org/p/chromium/issues/detail?id=724013) about it which we are planning to close soon.

### PR DESCRIPTION
**Pull request by mustaqahmed**
Adds coordinates to the list of properties of `pointercancel` that remain unchanged from the last dispatched event for the same `pointerid`.

Closes #463

***
<a href="https://pr-preview.s3.amazonaws.com/mustaqahmed/pointerevents/pull/473.html" title="Last updated on Jul 4, 2023, 3:41 PM UTC (4215ecf)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/pointerevents/473/300cca0...mustaqahmed:4215ecf.html" title="Last updated on Jul 4, 2023, 3:41 PM UTC (4215ecf)">Diff</a>

### CODE REVIEW

**Comment by mustaqahmed** [comment]
FYI: the third commit in my forked repo is not appearing here.  There should be 3 commits.

**Comment by mustaqahmed** [comment]
This is ready for review, please take a look.

**Comment by patrickhlauke** [comment]
sorry, that slipped me by. will review later today. thank you @mustaqahmed

File: index.html [hunk 1]
```diff
@@ -621,7 +621,7 @@ <h3>The <dfn data-dfn-for="GlobalEventHandlers" data-event-type="event">pointeru
...
+                <p>The values of the following properties of the {{GlobalEventHandlers/pointercancel}} event MUST match the values of the last dispatched pointer event with the same <code>pointerId</code>: <code>width</code>, <code>height</code>, <code>pressure</code>, <code>tangentialPressure</code>, <code>tiltX</code>, <code>tiltY</code>, <code>twist</code>, <code>altitudeAngle</code>, <code>azimuthAngle</code>, <code>pointerType</code>, <code>isPrimary</code> and the coordinates inherited from [[UIEVENTS]]. The <code>coalescedEvents</code> and <code>predictedEvents</code> lists in the {{GlobalEventHandlers/pointercancel}} event MUST be empty, and the event's {{Event/cancelable}} attribute MUST be false.</p>
```
Comment (patrickhlauke) [inline index.html]:
```suggestion
                <p>The values of the following properties of the {{GlobalEventHandlers/pointercancel}} event MUST match the values of the last dispatched pointer event with the same <code>pointerId</code>: <code>width</code>, <code>height</code>, <code>pressure</code>, <code>tangentialPressure</code>, <code>tiltX</code>, <code>tiltY</code>, <code>twist</code>, <code>altitudeAngle</code>, <code>azimuthAngle</code>, <code>pointerType</code>, <code>isPrimary</code>, and the coordinates inherited from [[UIEVENTS]]. The <code>coalescedEvents</code> and <code>predictedEvents</code> lists in the {{GlobalEventHandlers/pointercancel}} event MUST be empty, and the event's {{Event/cancelable}} attribute MUST be false.</p>
```

**Comment by mustaqahmed** [comment]
We have added a WPT for the PR #464.

**Pull request merged**